docs/v3/data-types.rst

```diff
@@ -2,12 +2,20 @@
 Data Types
 ==========
 
-Under construction.
+.. COMMENT TO BE REMOVED WHEN ONE IS ADDED
 
-.. toctree::
+   The following documents specify data types which are defined by the maintainers of
+   the Zarr specification. Being listed below does not imply that a data type is
+   required to be implemented by implementations.
+
+   toctree::
    :glob:
    :maxdepth: 1
    :titlesonly:
    :caption: Contents:
 
    data-types/*/*
+
+Currently, no data types are defined by the maintainers of
```

Comment:
This is a strange comment since there are data types defined, they are just listed elsewhere.

Reply:
:+1: I'll add a commit that tries moving the formal list to this subpage, otherwise it's unclear why it's even here.